README.md

```diff
@@ -2,10 +2,7 @@
 [![build status](https://travis-ci.org/bugsnag/bugsnag-ruby.svg?branch=master)](https://travis-ci.org/bugsnag/bugsnag-ruby)
 
 
-The Bugsnag exception reporter for Ruby gives you instant notification of exceptions
-thrown from your **[Rails](https://bugsnag.com/platforms/rails)**, **Sinatra**, **Rack** or **plain Ruby** app.
-Any uncaught exceptions will trigger a notification to be sent to your
-Bugsnag project.
+The Bugsnag exception reporter for Ruby gives you instant notification of exceptions thrown from your **[Rails](https://bugsnag.com/platforms/rails)**, **Sinatra**, **Rack** or **plain Ruby** app. Any uncaught exceptions will trigger a notification to be sent to your Bugsnag project.
```

Comment:
Sinatra/Rack/plain ruby should also be links. We also need docs for Sinatra as the installation method is slightly different to rack (in the docs, not in the readme).